src/draw/sw/blend/helium/lv_blend_helium.h

```diff
@@ -0,0 +1,1346 @@
+/**
+ * @file lv_blend_to_helium.h
+ *
+ */
+
+#ifndef LV_BLEND_TO_HELIUM_H
+#define LV_BLEND_TO_HELIUM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+//#include "../../../../lv_conf_internal.h"
+#include "lvgl.h"
+
```

Comment:
Let's use relative path instead. Include the whole "lvgl.h" seems unnecessary.

Reply:
Done